wurst/_handles/Ability.wurst

```diff
@@ -0,0 +1,106 @@
+package Ability
```

Comment:
I believe this package should be added to `Handles.wurst` auto import.

Reply:
Added